src/pipeline_edsl/pipeline_library.ml

```diff
@@ -22,6 +22,114 @@ module Input = struct
   let of_bam ?fragment_id ?sorted ~reference_build how s =
     (fragment_id, Of_bam (how, sorted, reference_build,  s))
   let fastq_sample ~sample_name files = Fastq {sample_name; files}
+
+  let tag_v0 = "Biokepi_input_v0"
+  let current_version_tag = tag_v0
+
+  let to_yojson =
+    let string s = `String s in
+    let option f o : Yojson.Safe.json = Option.value_map o ~default:`Null ~f in
+    let data_to_yojson =
+      function
+      | PE (r1, r2) ->
```

Comment:
I know this change happened in another PR, but PE -> Paired_end, etc, would make this a bit clearer to me.